server/src/main/java/com/google/musicanalysis/types/CommentLikes.java

```diff
@@ -0,0 +1,14 @@
+package com.google.musicanalysis.types;
+
+import java.io.Serializable;
+
+/** Groups together a comment with its likes */
+public class CommentLikes implements Serializable {
```

Comment:
nice name

Reply:
I would consider calling this "Comment" instead of "CommentLikes" because each comment "has" likes and "has" text content. You're already using it throughout your files with an associated variable name of "comments" or "commentsArray" as well